.github/workflows/python-app.yml

```diff
@@ -24,6 +24,10 @@ jobs:
         python-version: "3.11"
     - name: Install dependencies
       run: |
+        sudo apt-get update -qq
+        sudo add-apt-repository ppa:ubuntugis/ppa -y
```

Comment:
Adding "sudo add-apt-repository ppa:ubuntugis/ppa -y" helps a bit but not completely. It bumps up the GDAL version from 3.4.1 to 3.6.4 (which is from about a year ago)!

Reply:
Done